harness/benchmarks/gd/Simple.gd

```diff
@@ -1,46 +1,51 @@
 func benchmark_simple_add():
-    var a = 1
-    var b = 2
-    return a + b
+	var a = 1
```

Comment:
We need to setup our godot editors and IDE's to either use spaces or tabs for gdscript. Else we permanently have changes in those files.
Nothing to do with this particular PR but definitely something we have to look at sooner than later IMO

Reply:
I agree with this, on my side I got conflicts between intellij and godot editor. Would be simpler to make a PR to solve this issue.

Reply:
Ok for me